ui/src/notebooks/pipes/Query/index.ts

```diff
@@ -2,15 +2,13 @@ import {register} from 'src/notebooks'
 import View from './view'
 import './style.scss'
 
-export type Visibility = 'visible' | 'hidden'
-
 register({
   type: 'query',
   component: View,
   button: 'Custom Script',
   initial: {
-    resultsVisibility: 'visible',
-    resultsPanelHeight: 200,
+    panelVisibility: 'visible',
```

Comment:
this resizing is a weird concept in here. like, is it notebook specific? but it's not shared by all panels... and a panel in the query context is a child of the notebook panel, but in a visualization, it's the panel itself? most of my hangups come from not having a set of clear definitions around this

Reply:
Same same. I briefly considered moving it into meta or even its own context but did not because only some types would use it